l10n_br_account_payment_cobranca/models/account_payment_mode.py

```diff
@@ -0,0 +1,151 @@
+    # débito e crédito.
+    boleto_carteira = fields.Char("Carteira", size=3)
+    boleto_modalidade = fields.Char("Modalidade", size=2)
+    boleto_convenio = fields.Char("Codigo convênio", size=10)
+    boleto_variacao = fields.Char("Variação", size=2)
+    boleto_cnab_code = fields.Char("Código Cnab", size=20)
+    boleto_aceite = fields.Selection(
+        [("S", "Sim"), ("N", "Não")], string="Aceite", default="N"
+    )
+    boleto_type = fields.Selection(selection=[], string="Boleto")
+    boleto_especie = fields.Selection(
+        [
+            ("01", "DUPLICATA MERCANTIL"),
+            ("02", "NOTA PROMISSÓRIA"),
+            ("03", "NOTA DE SEGURO"),
+            ("04", "MENSALIDADE ESCOLAR"),
+            ("05", "RECIBO"),
+            ("06", "CONTRATO"),
+            ("07", "COSSEGUROS"),
+            ("08", "DUPLICATA DE SERVIÇO"),
+            ("09", "LETRA DE CÂMBIO"),
+            ("13", "NOTA DE DÉBITOS"),
+            ("15", "DOCUMENTO DE DÍVIDA"),
+            ("16", "ENCARGOS CONDOMINIAIS"),
+            ("17", "CONTA DE PRESTAÇÃO DE SERVIÇOS"),
+            ("99", "DIVERSOS"),
+        ],
+        string="Espécie do Título",
+        default="01",
+    )
+    boleto_protesto = fields.Char(
+        # [ Deixado manualmente, pois cada banco parece ter sua tabela.
+        # ('0', u'Sem instrução'),
+        # ('1', u'Protestar (Dias Corridos)'),
+        # ('2', u'Protestar (Dias Úteis)'),
+        # ('3', u'Não protestar'),
+        # ('7', u'Negativar (Dias Corridos)'),
+        # ('8', u'Não Negativar')
+        # ]
+        string="Códigos de Protesto",
+        default="0",
+    )
+    boleto_protesto_prazo = fields.Char("Prazo protesto", size=2)
+    gera_nosso_numero = fields.Boolean(
+        string="Gerar nosso número?",
+        help="Dependendo da carteira, banco, etc. "
+        "O nosso número pode ser gerado pelo banco.",
+        default=True,
+    )
+    default_account_id = fields.Many2one(
+        comodel_name="account.account",
+        string="Conta Padrão",
+        help="Conta padrão para recebimentos",
+    )
+    default_tax_account_id = fields.Many2one(
+        comodel_name="account.account",
+        string="Conta Padrão para Taxas Bancárias",
+        help="Conta padrão para recebimentos de Taxas Bancárias",
+    )
+    product_tax_id = fields.Many2one(
+        comodel_name="product.product", string="Taxa Adicional"
+    )
+    tax_account_id = fields.Many2one(
+        comodel_name="account.account",
+        string="Conta Padrão",
+        help="Conta padrão para Taxa",
+    )
+    operation_type = fields.Selection(selection_add=[("3", "Cobrança")])
+    cnab_sequence_id = fields.Many2one(
+        comodel_name='ir.sequence',
+        string=u'Sequencia do CNAB')
+
+
+    @api.onchange("product_tax_id")
+    def _onchange_product_tax_id(self):
+        if not self.product_tax_id:
+            self.tax_account_id = False
+
+    @api.constrains("product_override")
```

Comment:
WARNING db odoo.models: method account.payment.mode._constrains_product_override: @constrains parameter 'product_override' is not a field name 
Não encontrei esse campo na versão 8, faltou algo ? Alguma herança ?
cc @sadamo @mileo

Reply:
Não localizei aqui também.

https://github.com/OCA/l10n-brazil/pull/700/files?file-filters%5B%5D=.py 

Deixa comentado por hora.